gnc/.sub8_trajectory_generator/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(${PROJECT_NAME})
 find_package(catkin REQUIRED COMPONENTS roscpp geometry_msgs sub8_msgs cmake_modules)
 find_package(OMPL REQUIRED)
 find_package(Boost REQUIRED)
-find_package(Eigen REQUIRED)
+find_package(EIGEN REQUIRED)
```

Comment:
It doesn't matter since this is not currently being compiled, but this was supposed to be `Eigen3`.